src/dict.c

```diff
@@ -318,6 +318,72 @@ int dictShrink(dict *d, unsigned long size) {
     return _dictResize(d, size, NULL);
 }
 
+/* Helper function for `dictRehash` and `dictBucketRehash` which rehashes all the keys
+ * in a bucket at index `idx` from the old to the new hash HT. */
+void _rehashEntriesInBucketAtIndex(dict *d, uint64_t idx) {
```

Comment:
```suggestion
static void rehashEntriesInBucketAtIndex(dict *d, uint64_t idx) {
```
The underscore is used in other functions to allow two functions with the same name, I don't think we need to do that here.